README.md

```diff
@@ -2,17 +2,24 @@
 
 This module implements the [`rdk:component:board` API](https://docs.viam.com/components/board/#api) and [`rdk:component:servo` API](https://docs.viam.com/components/servo/#api)
 
-The following models are provided in this module:
+This module provides the following models to access GPIO functionality (input, output, PWM, power, serial interfaces, etc.):
 
-* `viam:raspberry-pi:rpi5` - Configure a Raspberry Pi 5 board to access GPIO functionality: input, output, PWM, power, serial interfaces, etc.
+* `viam:raspberry-pi:rpi5` - Configure a Raspberry Pi 5 board 
 * `viam:raspberry-pi:rpi4` - Configure a Raspberry Pi 4 board
 * `viam:raspberry-pi:rpi3` - Configure a Raspberry Pi 3 board
 * `viam:raspberry-pi:rpi2` - Configure a Raspberry Pi 2 board
 * `viam:raspberry-pi:rpi1` - Configure a Raspberry Pi 1 board
 * `viam:raspberry-pi:rpi0` - Configure a Raspberry Pi 0 board
 * `viam:raspberry-pi:rpi0_2` - Configure a Raspberry Pi 0 2 W board
+
+This module also provides a servo model:
+
 * `viam:raspberry-pi:rpi-servo` - Configure a servo controlled by the GPIO pins on the board. Note this model is not supported on the rpi5 board.
 
+Note that Raspberry Pi 0, 1, 2 and 3 are supported as GPIO peripherals but are not capable of running `viam-server` themselves.
+You can use these boards with Viam by connecting them to another computer or SBC.
+RPi 0 2W, 4, and 5 can run `viam-server`.
```

Comment:
Hmm okay. I'm talking to Abe about what to put in docs about armv7l/32 bit support. We had an AI chat question about which Pis are supported and the bot just rattled off this whole list. How about this?
```suggestion
All of the Pis above are supported as [board components](https://docs.viam.com/operate/reference/components/board/), but some older models are not capable of running `viam-server`--see [Set up a computer or SBC](https://docs.viam.com/operate/get-started/setup/) for `viam-server` system requirements.
```
Or if you think this is overkill or actually increases confusion, I can remove this part from the PR (still think formatting change above adds clarity) and put it as hidden content in the docs instead.

Reply:
I'm okay with this version.

Also, this part of the readme isn't as visible to our platform users as the inline docs. By the point they try to run viam-server and it fails on a raspberry pi 0 or 1 - they'll know that they can't get a board component. This copy reflects that I think.